hal-sim/setup.py

```diff
@@ -39,26 +39,35 @@
 with open(join(setup_dir, 'README.rst'), 'r') as readme_file:
     long_description = readme_file.read()
 
-setup(
-    name='robotpy-hal-sim',
-    version=__version__,
-    description='WPILib HAL layer for simulations',
-    long_description=long_description,
-    author='Peter Johnson, Dustin Spicuzza',
-    author_email='robotpy@googlegroups.com',
-    url='https://github.com/robotpy/robotpy-wpilib',
-    keywords='frc first robotics hal can',
-    packages=['hal_impl'],
-    install_requires='robotpy-hal-base==' + __version__, # is this a bad idea?
-    license="BSD License",
-    classifiers=[
-        "Development Status :: 5 - Production/Stable",
-        "Intended Audience :: Developers",
-        "Intended Audience :: Education",
-        "License :: OSI Approved :: BSD License",
-        "Operating System :: OS Independent",
-        "Programming Language :: Python :: 3.4",
-        "Programming Language :: Python :: 3.5",
-        "Topic :: Scientific/Engineering"
-    ]
-    )
+if __name__ == '__main__':
+    if len(sys.argv) > 1 and sys.argv[1] == 'install':
+        # setuptools doesn't seem to have a nice way of running checks before
+        # an install script runs, so I'm just going to do this and hope it works
+
+        # NOTE: may have false positives, but it should work well enough
+        if exists('/etc/natinst/share/scs_imagemetadata.ini'):
```

Comment:
Can you add the 'same' check to hal-roborio/setup? `if not exists(...`

Reply:
I can't say I can imagine anyone accidentally installing robotpy-hal-roborio on their computer, really.

Reply:
it...could happen. really only for devs who are installing from source